src/routes/details/[id]/+page.svelte

```diff
@@ -0,0 +1,26 @@
+<script>
+	export let data;
+	let anime = data.anime;
+
+	const isEmpty = (obj) => {
+		return Object.keys(obj).length === 0;
+	};
+
+	$: animeSynopsis = anime?.synopsis ? anime.synopsis : 'Pas de synopsis disponible';
```

Comment:
Pas evident à lire, mais on peut simplifier à:
```suggestion
$: animeSynopsis =anime?.synopsis ?? 'Pas de synopsis disponible';
```

Reply:
Ah top, je ne connaissais pas cette astuce, le deuxième point d'interrogation permet d'attribuer la variable `anime.synopsis` implicitement dans le cas où elle existe ?